### ISSUE TITLE
Searching by power/toughness breaks when not specified as power/toughness in the XML

### ISSUE THREAD

**Issue by DragoonX6** [Cockatrice/Cockatrice#5109]
In the WIXOSS plugin only power is specified, since WIXOSS does not have
the concept of toughness. Instead, power of SIGNI (creatures) both
functions as power and toughness as the same time. We specify it as
e.g. `<pt>3000</pt>` in the XML. This, however, causes cards to be
unsearchable by power and/or toughness as anything you specify as the
filter value will return no results. I suspect this is because the
values are parsed in the format of `P/T`, though I haven't bothered to
check the code.

Please handle a singular value in the `pt` property to mean `P/T`.
This should help plugins for other games as well as they might be
specifying power in the same way.

**Comment by ebbit1q** [comment]
I can't reproduce your issue, searching for power=3000 results in the card with `<pt>3000</pt>` for me

**Comment by DragoonX6** [comment]
Interestingly enough searching for `pt=3000` in the card name box does work.
But when I try to do it through the power filter, no results are returned.

Is it possible to make it work through the filters?

**Comment by ebbit1q** [comment]
I don't know, we almost removed the filter widget, the text filters were meant to replace them

**Comment by tooomm** [comment]
>the text filters were meant to replace them

Yes, the idea was to let them exist next to each other for a while for users to get familiar with the new and more powerful text filter which works the same as in other places like Scryfall and users are used to.
I would not mind if we remove it after the next full release.

**However, the widget allows better to switch between filters and preserves them.
In order to improve that for the text filters in the search bar, there needs to be e.g. a history of last ~3 searches.**
Ideally, there should be a way to star favorites searches and pin/preserve them for easy re-use (even after restart, something the widget is also not capable of. I think there is even a "save filter" request somewhere).

**Comment by ebbit1q** [comment]
the addition of text filters marked the "save filter" request as fixed as it allows users to save filters, a search history, or integrating the saving of searches in client would be a neat feature but a separate request I think

**Comment by DragoonX6** [comment]
I'm not sure how I feel about the potential removal of the search widget, I'll have to take a deeper look at the search syntax and how user friendly it is to create huge and complicated searches. I'm thinking the search bar might have to become multi line to enhance user friendliness.

**Comment by ebbit1q** [comment]
that is also a good idea

**Comment by tooomm** [comment]
>the addition of text filters marked the "save filter" request as fixed as it allows users to save filters

You mean one can copy a search to a text file and save it outside of Cockatrice? Well, I would not call that a solution.
In https://github.com/Cockatrice/Cockatrice/issues/4955 the request came up again and we discussed further.

>a search history, or integrating the saving of searches in client would be a neat feature but a separate request I think

https://github.com/Cockatrice/Cockatrice/issues/4955 👍

<br>

>I'm not sure how I feel about the potential removal of the search widget, I'll have to take a deeper look at the search syntax and how user friendly it is to create huge and complicated searches. I'm thinking the search bar might have to become multi line to enhance user friendliness.

I highly recommend you to give it a try!
It works like what you know from other famous card pages like [Scryfall](https://scryfall.com/) when filtering results and chaining search parameters for finding cards.

Also check the syntax examples and hints!
<img width="417" alt="image" src="https://github.com/user-attachments/assets/cc2f755a-683c-424a-a1fe-7d0944f7381c">
